examples/spark2/src/test/scala/org/apache/carbondata/examplesCI/RunExamples.scala

```diff
@@ -129,4 +129,8 @@ class RunExamples extends QueryTest with BeforeAndAfterAll {
   test("HiveExample") {
     HiveExample.exampleBody(spark, TestQueryExecutor.warehouse)
   }
+
+  test("DirectSQLExample") {
+    DirectSQLExample.main(null)
```

Comment:
Please refer to 'CarbonSessionExample.exampleBody(spark)', use 'spark' created in RunExamples, don't re-create one in DirectSQLExample.

Reply:
Thanks for your suggesion @zzcclp, done.